odoo/modules/registry.py

```diff
@@ -52,6 +53,7 @@
     'assets': ('assets', 'templates.cached_values'),
     'templates': ('templates', 'templates.cached_values'),
     'routing': ('routing', 'routing.rewrites', 'templates.cached_values'),
+    'groups': ('groups',),
```

Comment:
I may add 'templates.cached_values' here

Reply:
Actually since it implies _has_group, I think it may be almost all caches, so default should be added too maybe. 

For templates, it may have an impact for _get_view_cache but is was already the case before this change so not sure